src/AutoMapper/IMappingOperationOptions.cs

```diff
@@ -22,7 +22,7 @@ public interface IMappingOperationOptions
         /// <summary>
         /// Disable the cache used to re-use destination instances based on equality
         /// </summary>
-        bool DisableCache { get; set; }
+        bool PreserveReferences { get; set; }
```

Comment:
Let's move this up to the TypeMap config level. Disabling is per operation but enabling is per map.